R/object_usage_linter.R

```diff
@@ -5,11 +5,12 @@
 #'
 #' @param interpret_glue If `TRUE`, interpret [glue::glue()] calls to avoid false positives caused by local variables
 #' which are only used in a glue expression.
+#' @param skip_with A logical. If `TRUE` (default), code in `with()` expressions will be skipped.
```

Comment:
lets mention this is passed along to the `skipWith` argument to remove any ambiguity